usage/keyboard-shortcuts.md

```diff
@@ -61,4 +66,4 @@ The FreeTube client can also be controlled with shortcuts you might already be f
 | Alt + Left Arrow                               | Go back one page                       |
 | Alt + Right Arrow                              | Go forward one page                    |
 
-If you'd like to see even more keyboard shortcuts, consider creating a [feature request](https://github.com/FreeTubeApp/FreeTube/issues/new?assignees=&labels=enhancement&template=feature_request.yaml&title=%5BFeature+Request%5D%3A+).
+If you'd like to see even more keyboard shortcuts, consider creating a [feature request](https://github.com/FreeTubeApp/FreeTube/issues/new?assignees=&labels=enhancement&template=feature_request.yaml&title=%5BFeature+Request%5D%3A+).
```

Comment:
Is there a change here? I can't spot it

Also waiting on release to merge (ping me though coz I definitely will forget